src/etc/test-float-parse/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+[package]
+name = "test-float-parse"
+version = "0.1.0"
+edition = "2018"
+publish = false
+
+[workspace]
+
+[dependencies]
+rand = "0.4"
+
+[lib]
+name = "test_float_parse"
+path = "lib.rs"
+
+[[bin]]
+name = "few-ones"
+path = "few-ones.rs"
+
+[[bin]]
+name = "huge-pow10"
+path = "huge-pow10.rs"
+
+[[bin]]
+name = "long-fractions"
+path = "long-fractions.rs"
+
+[[bin]]
+name = "many-digits"
+path = "many-digits.rs"
+
+[[bin]]
+name = "rand-f64"
```

Comment:
You shouldn't need to add all these explicitly, cargo will pick them up automatically as long as you don't set `autobins = false`.

Reply:
[Refactored](https://github.com/Alexhuszagh/rust/blob/870ed2163282b2e5fa99585e6c28c916e4861646/src/etc/test-float-parse/Cargo.toml) the crate to automatically find bins.